### PR TITLE
Add context menu action to play current selection

### PR DESCRIPTION
**Pull request by thdg**
Added a context menu action to play the current selection.

It is not possible to create a context menu item in a "content" so I needed to add a background script that runs once on startup.
It sends a message to the correct tab and "presses" play. The play function already plays the selected text if there is a text selected so that is all.

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Add action to context menu
+chrome.contextMenus.create({
+    id: "play-action",
+    title: "Splia",
```
Comment (staffanru) [inline background.js]: ``
```suggestion
    title: "WebRICE - Splia",
```

Comment (staffanru) [reply background.js]: just to make it clear that it is webrice

File: background.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    contexts: ["selection"],
+  },
+);
+
+// Handler for centext menu
+const onClick = (info, tab) => {
+  if (info.menuItemId == "play-action") {
+    chrome.tabs.sendMessage(tab.id, {action: "play"});
```
Comment (staffanru) [inline background.js]:
```suggestion
    chrome.tabs.sendMessage(tab.id, {
      receiver: "content_script",
      command: "play",
    });
```

Comment (staffanru) [reply background.js]: with this change the added listener in content is not necessary

File: content/content.js [hunk 3]
```diff
@@ -164,3 +164,12 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
...
 });
+
+/**
+ * Recieve messages
+ */
+chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
+  if (request.action === "play") {
+    play();
+  }
+});
```
Comment (staffanru) [inline content/content.js]: This can be removed if we change the background script, there is a listener already

Comment (staffanru) [inline content/content.js]:
```suggestion
```

**Review by staffanru — COMMENTED** [review]
Nice work!

I added some suggestions to use the existing message system.

File: background.js (hunk 1 above)
Comment (thdg) [reply background.js]: I wondered about that, however the WebRICE icon is displayed beside the text so was not sure if this is needed.
Also this does not seem to be the convention (multiple items from the same extension will look to similar maybe).
Maybe a more descriptive text? "Lesa upp valinn texta"

File: background.js (hunk 2 above)
Comment (thdg) [reply background.js]: Nice

File: background.js (hunk 1 above)
Comment (staffanru) [reply background.js]: I had compared it to the read aloud one. But what it says is
"Read aloud selected text", the name is literally the part of the sentence that makes sense. 
I like "Lesa upp valin texta"

Comment (G-Thor) [reply background.js]: Seconded. "Lesa upp valinn texta" is best imo

**Comment by judyfong** [comment]
might solve #1

**Comment by thdg** [comment]
Added the changes and a more descriptive name for the action.
This was indeed added to solve #1

**Review by staffanru — APPROVED** [review]

**Pull request merged**